<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newUser</class>
 <widget class="QDialog" name="newUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>creatAccount</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="NameText">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>120</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(151, 0, 111);
border-color: rgb(151, 0, 111);
padding : 2px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="الاسم">الاسم</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 111);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>ادخل كلمة مرور المدير :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>461</width>
     <height>251</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>461</width>
     <height>251</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 111);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLable">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 111);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>الاسم :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="adminpassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(151, 0, 111);
border-color: rgb(151, 0, 111);
padding : 2px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>كلمة مرور المدير</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 111);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>انشاء حساب</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLable">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 111);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>كلمة المرور :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(151, 0, 111);
border-color: rgb(151, 0, 111);
padding : 2px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>كلمة المرور</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newAccount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(151, 0, 111);
color: rgb(255, 255, 255);
font: 10pt &quot;Bernard MT Condensed&quot;;
</string>
   </property>
   <property name="text">
    <string>انشاء حساب</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>NameText</zorder>
  <zorder>adminLable</zorder>
  <zorder>nameLable</zorder>
  <zorder>adminpassword</zorder>
  <zorder>label_2</zorder>
  <zorder>passwordLable</zorder>
  <zorder>passwordText</zorder>
  <zorder>newAccount</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
